configuration/concepts/void_examinations.csv

```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1600
+18e738c6-a8b9-4f42-a87f-a2d7f291aca3,TRUE,,Examination,Examen,Examination,Examen,,,Misc,Coded,,Normal;Anormal,,,,,,,,,
```

Comment:
I guess you just need the `uuid` and `void/retire`.

Reply:
@Ruhanga do I remove the other fields?

Reply:
To something like below ?

`18e738c6-a8b9-4f42-a87f-a2d7f291aca3,TRUE,,,,,,,,,Coded,,,,,,,,,,,`

Reply:
> the uuid and void/retire fields.

Does it not work, the suggestion above, something along the lines 

`18e738c6-a8b9-4f42-a87f-a2d7f291aca3,TRUE,,,,,,,,,,,,,,,,,,,,,,,,,,,......`?